File: src/main/java/sk/tuke/gamestudio/game/bricksbreaking/stovka/game/Health.java
package sk.tuke.gamestudio.game.bricksbreaking.stovka.game;

public class Health {

    private int health = 5;

    public void record(int removedTiles) {
        if (removedTiles == 1) {
            health--;
        }
    }

    public boolean isAlive() {
        return health > 0;
    }

    public int getHealth() {
        return health;
    }
}

File: web/src/game/board/health.ts

export class Health {

    private health = 5;

    record(removeTiles: number) : void {
        if (removeTiles <= 1) {
            this.health -= 1;
        }
    }

    isAlive() : boolean {
        return this.health > 0;
    }

    getHealth() : number {
        return this.health;
    }

}
